modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/io/LogEntryJsonWriterTest.java

```diff
@@ -114,4 +119,42 @@ public void testArrayInExtendedInfo() throws Exception {
         json.has("extended").properties(1).has("params").isArray().contains("a simple string");
     }
 
+    @Test
+    public void testMapInExtendedInfo() throws Exception {
+        Map<String, ExtendedInfo> infos = new HashMap<>();
+
+        HashMap<String, Serializable> infoMap = new HashMap<>();
+        infoMap.put("String","abcde");
+        Date now = new Date();
+        infoMap.put("Date", now);
+        infoMap.put("Boolean",false);
+        infoMap.put("Integer",1);
+        infoMap.put("Double",2.0);
+        infos.put("params", ExtendedInfoImpl.createExtendedInfo(infoMap));
```

Comment:
```suggestion
        var infoMap = Map.of("String", "abcde", "Date", new Date(), "Boolean", false, "Integer", 1, "Double", 2.0);
        infos.put("params", ExtendedInfoImpl.createExtendedInfo((Serializable) infoMap));
```

Reply:
@NourNuxeo if we do that it wont' be directly backportable. And for a test it isn't as readable.

Reply:
didn't do